Ladybird/ImageCodecPluginLadybird.cpp

```diff
@@ -14,30 +14,10 @@ namespace Ladybird {
 
 ImageCodecPluginLadybird::~ImageCodecPluginLadybird() = default;
 
-static Optional<Web::Platform::DecodedImage> decode_image_with_qt(ReadonlyBytes data)
-{
-    auto image = QImage::fromData(data.data(), static_cast<int>(data.size()));
-    if (image.isNull())
-        return {};
-    image = image.convertToFormat(QImage::Format::Format_ARGB32);
-    auto bitmap = MUST(Gfx::Bitmap::create(Gfx::BitmapFormat::BGRA8888, Gfx::IntSize(image.width(), image.height())));
-    for (int y = 0; y < image.height(); ++y) {
-        memcpy(bitmap->scanline_u8(y), image.scanLine(y), image.width() * 4);
-    }
-    Vector<Web::Platform::Frame> frames;
-
-    frames.append(Web::Platform::Frame {
-        bitmap,
-    });
-    return Web::Platform::DecodedImage {
-        false,
-        0,
-        move(frames),
-    };
-}
-
-static Optional<Web::Platform::DecodedImage> decode_image_with_libgfx(ReadonlyBytes data)
+Optional<Web::Platform::DecodedImage> ImageCodecPluginLadybird::decode_image(ReadonlyBytes data)
 {
+    // Note: We only ask LibGfx to attempt to decode the image. While we could ask Qt too if LibGfx fails,
+    // that'd be cheating, and an image failing to load should be motivation to improve LibGfx/LibWeb :^)
```

Comment:
This comment seems superfluous. :)

Reply:
Also I think you can remove the `QImage` header now?

Reply:
Yep, done :) The comment was mainly to prevent someone getting the idea again :P